src/Employment/WorkContract_v0.1.py

```diff
@@ -0,0 +1,339 @@
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class WorkContractRequest(CamelCaseModel):
+    pass
+
+
+class EmployerInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The official name of the employer",
+        examples=["Staffpoint Oy"],
+    )
+    businessID: str = Field(
+        ...,
+        title="Business ID",
+        max_length=250,
+        description="The business ID of the employer",
+        examples=["2492090-1"],
+    )
+    street_name: str = Field(
+        ...,
+        title="Street Name",
+        max_length=40,
+        description="The street name of the employer contact address",
+        examples=["Ruoholahdenkatu 17 A 6"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=10,
+        description="The postal code of the employer address",
+        examples=["00180"],
+    )
+    city: str = Field(
+        ...,
+        title="City",
+        max_length=40,
+        description="The city of the employer address",
+        examples=["Helsinki"],
+    )
+    signature_date: date = Field(
+        ...,
+        title="Signature Date",
+        description="The date of the employer signature for the work contract",
+        examples=[datetime.datetime(2023, 8, 25)],
+    )
+
+
+class EmployeeInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The name of the employee",
+        examples=["Tom Williams"],
+    )
+    street_address: str = Field(
+        ...,
+        title="Street Address",
+        max_length=40,
+        description="The street name of the employee contact address",
+        examples=["49 Featherstone Street"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=10,
+        description="The postal code of the employee address",
+        examples=["EC1Y 8SY"],
+    )
+    city: str = Field(
+        ...,
+        title="City",
+        max_length=40,
+        description="The city of the employee address",
+        examples=["London"],
+    )
+    country: Optional[str] = Field(
+        None,
+        title="Country",
+        max_length=3,
+        min_length=3,
+        description="The country code of the employee address in Alpha-3 format",
+        examples=["GBR"],
+    )
+    signature_date: date = Field(
+        ...,
+        title="Signature Date",
+        description="The date of the employee signature for the work contract",
+        examples=[datetime.datetime(2023, 8, 27)],
+    )
+
+
+class TermsOfWork(CamelCaseModel):
+    employment_start: date = Field(
+        ...,
+        title="Employment Start",
+        description="The start date of the employment",
+        examples=[datetime.datetime(2023, 11, 7)],
+    )
+    employment_end: date = Field(
```

Comment:
I realized that if this is a mandatory field, then you can only have fixed term contracts which was not the idea. Checked with Henna and updated this to an optional field, so please change accordingly.